src/e2e-test/java/io/cdap/plugin/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
```

Comment:
What is this file being used for ?

Reply:
This file is used to remove checkstyle errors and the time of build